internal/models/authenticator.go

```diff
@@ -0,0 +1,59 @@
+package models
+
+import (
+	"fmt"
+
+	"github.com/unicitynetwork/aggregator-go/pkg/api"
+)
+
+// Authenticator represents the authentication data for a commitment
+type Authenticator struct {
```

Comment:
why this (third) struct is needed? Seems api.Authenticator and AuthenticatorBSON to be enough

Reply:
We didn't add it, it was just moved to separate file. Looks like design choice to separate api and domain structs. Since it already exists, even though all the fields are the same, it's not worth refactoring IMO.